templates/federated_reporting/distributed_cleanup.py

```diff
@@ -120,11 +128,16 @@ def interactive_setup_feeder(hub, email, fr_distributed_cleanup_password):
         sys.exit(1)
 
 
-def interactive_setup():
-    fr_distributed_cleanup_password = "".join(random.choices(string.printable, k=20))
-    admin_pass = getpass(
-        prompt="Enter admin password for superhub {}: ".format(socket.getfqdn())
-    )
+def interactive_setup(force_interactive=False):
+    fr_distributed_cleanup_password = "".join(random.choices(string.digits + string.ascii_letters, k=20))
```

Comment:
What is this fr_distributed_cleanup_password bit with random choices?

Reply:
This is the password for the account created and used later by this utility. The password is saved via cf-secret, encrypted to only the superhub.